src/clue.js

```diff
@@ -1,28 +1,138 @@
 // ITERATION 1
 
-// Suspects Array
-
-const suspectsArray = [];
-
+/* Suspects Array  
+Clue has three different card types: suspects, rooms, and weapons.
+
+Let's create an array for every card type. These arrays should be named suspectsArray, weaponsArray, roomsArray. 
+Each array will contain objects, where each object represents one card.
+
+All of the information you need about suspects, rooms, and weapons can be found on the DATA.md file.
+*/
+
+
+const suspectsArray = [
+     
```

Comment:
para objetos usa esta sintaxis
const suspectsArray = [
        firstName: 'Jack',
        lastName: 'Mustard',
        occupation: 'Retired Football player',
        age: 62,
        description: 'He is a former football player who tries to get by on his former glory',
        image: 'https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg',
        color: 'yellow'
]